### PR TITLE
RC: Throughput section changes

### PR DESCRIPTION
**Pull request by cmilesb**
Ticket: DOC-3492
Staging link: https://docs.redis.com/staging/DOC-3492/rc/databases/create-database#throughput

### CODE REVIEW

File: content/rc/databases/create-database.md [hunk 1]
```diff
@@ -143,10 +143,12 @@ Throughput is the amount of operations a database can handle over a certain peri
...
 - **Command complexity**: O(N) and O(log(N)) commands will take more time than O(1) commands, and will affect throughput accordingly.
-- **Key and value sizing**: If your database's keys and values are very large, setting and reading those keys may take more time and affect throughput.
+- **Key and value sizing**: If your database's keys and values are very large, setting and reading those keys may take more time and affect throughput. If these are lower than the typical workload, the actual throughput might be higher than expected.
 - **Replication**: Using [multi-zone replication]({{<relref "rc/databases/configuration/high-availability">}}) affects throughput as each write operation is executed asynchronously in each zone.
+- **Security**: Some security options, such as [transport layer security]({{< relref "/rc/security/database-security/tls-ssl" >}}), may affect throughput.
+- **Number of client connections**: The number of client connections affects throughput. As connections to the database increase, the throughput decreases.
```
Comment (AlonMagrafta) [inline content/rc/databases/create-database.md:151]: That is incorrect. Number of client connections can affect performance both ways (higher throughput / lower throughput and same for latencies).

File: content/rc/databases/create-database.md [hunk 2]
```diff
@@ -143,10 +143,12 @@ Throughput is the amount of operations a database can handle over a certain peri
...
 
-Your actual throughput may not match the throughput you set when you create your database. The following things can affect your database's throughput:
+We assume a typical workload that includes a different mix of commands and an average key and value size of 1KB. Therefore, your actual throughput be higher or lower than the throughput you set when you create your database. The following properties can affect your database's throughput:
 - **Command complexity**: O(N) and O(log(N)) commands will take more time than O(1) commands, and will affect throughput accordingly.
-- **Key and value sizing**: If your database's keys and values are very large, setting and reading those keys may take more time and affect throughput.
+- **Key and value sizing**: If your database's keys and values are very large, setting and reading those keys may take more time and affect throughput. If these are lower than the typical workload, the actual throughput might be higher than expected.
```
Comment (AlonMagrafta) [inline content/rc/databases/create-database.md:148]: @NoamSternRedis I'm not sure we wanna say that

**Review by AlonMagrafta — CHANGES_REQUESTED** [review]

File: content/rc/databases/create-database.md (hunk 1 above)
Comment (cmilesb) [reply content/rc/databases/create-database.md:151]: In that case, how about this:
```suggestion
- **Number of client connections**: The number of client connections affects throughput. If the number of client connections is higher or lower than expected, throughput may be affected accordingly.
```

Comment (AlonMagrafta) [reply content/rc/databases/create-database.md:151]: mmm still not what we want to say.
I suggest something like:
- **Number of client connections**: The number of client connections affects throughput. Increasing or decreasing the number of client connections can result in higher or lower throughput.

**Review by AlonMagrafta — DISMISSED** [review]

File: content/rc/databases/create-database.md (hunk 1 above)
Comment (cmilesb) [reply content/rc/databases/create-database.md:151]:
```suggestion
- **Number of client connections**: The number of client connections affects throughput. Increasing or decreasing the number of client connections can result in higher or lower throughput.
```

**Comment by AlonMagrafta** [comment]
Other than my comment to @NoamSternRedis - LGTM now :)

File: content/rc/databases/create-database.md [hunk 3]
```diff
@@ -143,10 +143,12 @@ Throughput is the amount of operations a database can handle over a certain peri
 
 Databases with search and query enabled use the number of shards to determine throughput. To determine how many shards you need for your database, use the [sizing calculator](https://redis.com/modules/redis-search/redisearch-sizing-calculator/).
 
-Your actual throughput may not match the throughput you set when you create your database. The following things can affect your database's throughput:
+We assume a typical workload that includes a different mix of commands and an average key and value size of 1KB. Therefore, your actual throughput be higher or lower than the throughput you set when you create your database. The following properties can affect your database's throughput:
```
Comment (cmilesb) [inline content/rc/databases/create-database.md:146]:
```suggestion
We assume a typical workload that includes a different mix of commands and an average key and value size of 1KB. Therefore, your actual throughput may be higher or lower than the throughput you set when you create your database. The following properties can affect your database's throughput:
```

**Review by mich-elle-luna — APPROVED** [review]
Thank you, just one suggestion for clarity, to qualify "these" and align the word (smaller versus larger) with the previous sentence. But, it is up to you if that makes sense.

**Pull request merged**